app/views/films/index.html.erb

```diff
@@ -101,179 +66,12 @@
-
-<!--              <div class="movies_image">-->
-<!--                <div class="hover">-->
-<!--                  <i class="fa-solid fa-star"></i>-->
-<!--                  <p class="rate"><%#= movie['vote_average'] %></p>-->
-<!--                  <p class="categories"><%#= movie['genres'] %></p>-->
-                  <%#= button_to 'Add to Database', add_movie_path(movie_id: movie['_id']), method: :post, :class => 'btn btn-primary' %>
-
-                  <%#= form_with(url: add_movie_path(movie_id: movie['_id']), method: :post) do |f| %>
-                    <%#= f.hidden_field :title, value: movie['title'] %>
-                    <%#= f.hidden_field :rating, value: movie['vote_average'] %>
-                    <%#= f.hidden_field :genre, value: movie['genres'] %>
-                    <%#= f.hidden_field :year, value: movie['release_date'] %>
-                    <%#= f.hidden_field :picture, value: movie['poster_path'] %>
-                    <%#= f.submit 'Add to Database', class: 'btn' %>
-                  <%# end %>
-
-<!--                </div>-->
-                <%#= image_tag movie['poster_path'] %>
-<!--              </div>-->
-<!--              <div class="text">-->
-<!--                <p class="name"><%#= movie['title'] %></p>-->
-<!--                <p class="year"><%#= movie['release_date'] %></p>-->
-<!--              </div>-->
-<!--            </div>-->
-
-          <%# end %>
-<!--        </div>-->
-
-      <%# end %>
-
-
-<!--    </section>-->
-<hr>
-<caption><h1>New Movies From Api except Database</h1></caption>
-<table class="table table-bordered">
-  <thead>
-  <tr>
-    <th scope="col">#</th>
-    <th scope="col">Title</th>
-    <th scope="col">Release Date</th>
-    <th scope="col">Vote Average</th>
-    <th scope="col">Genres</th>
-    <th scope="col">Image</th>
-    <th scope="col">Actions</th>
-  </tr>
-  </thead>
-  <tbody>
-  <% @new_movies.in_groups_of(5, false) do |row| %>
-    <% row.compact.each_with_index do |movie, index| %>
-      <tr>
-        <th scope="row"><%= index + 1 %></th>
-        <td><%= movie['title'] %></td>
-        <td><%= movie['release_date'] %></td>
-        <td><%= movie['vote_average'] %></td>
-        <td><%= movie['genres'] %></td>
-        <td><%= image_tag movie['poster_path'], style: 'max-width: 50px; height: auto;' %></td>
-        <td>
-
-          <%= form_with(url: add_movie_path(movie_id: movie['_id']), method: :post) do |f| %>
-            <%= f.hidden_field :title, value: movie['title'] %>
-            <%= f.hidden_field :rating, value: movie['vote_average'] %>
-            <%= f.hidden_field :genre, value: movie['genres'] %>
-            <%= f.hidden_field :year, value: movie['release_date'] %>
-            <%= f.hidden_field :picture, value: movie['poster_path'] %>
-            <%= f.submit 'Add to Database', class: 'btn' %>
-          <% end %>
-        </td>
-      </tr>
-    <% end %>
-  <% end %>
-  </tbody>
-</table>
-
-
-
-
-
-
-
 </body>
 
 </html>
 
```

Comment:
lot of space issues and extra space un needed here please clear up this index.hmtl.erb view